jwql/tests/test_protect_module.py

```diff
@@ -13,13 +13,19 @@
     These tests can be run via the command line (omit the -s to
     suppress verbose output to stdout):
 
+    !!!WARNING IF TESTING LOCALLY!!!
+    These tests will call protect_module.py::lock_module.py and will sent emails to jwql.stsci.edu!
+    Please locally change the `message['To']` recipient to an appropriate address while testing to
+    avoid spamming/confusing the team.
+
+
```

Comment:
I like these changes, but I think this warning isn't sufficient to prevent regularly spamming the jwql list.  Can we make the email address configurable instead of hard-coded, and change it to a null address for testing?   Maybe make it a constant value at the top of the protect_module file? Or in the general jwql constants module?

Reply:
Looks good now! Thanks for the fix.